### PR TITLE
TRC-43- Add Global `Colors` Theme

### PR DESCRIPTION
**Pull request by mmcbride2929**
## TL/DR
This story involves creating a global color theme to keep our project's theme consistent.

## Overview of Changes

`Colors.scss`
- Declare our color scheme with a light & dark mode.

`Global.scss`
- Add `Global.scss` and import our `Colors.scss` to make declared colors available globally

`App.scss`
- Import `Global.scss` into projects main css file. 

`GlobalDecorator.tsx `
- Wraps stories with the global css file that we have imported.

`preview.js`
- Configures the decorator that wraps each Storybook story, giving the story access to the global colors without having to import them in each story individually.

`read.me`
- Updated docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,24 @@ In order to import our SVGs as React components, we uuse the ReactComponent impo
...
-=======
+### Global Theme / Color Scheme
+
+# Defining the Color Theme
+
+Create a `Colors.scss` file to define a set of colors that will be used throughout the project. Use the `:root` pseudo-class to define the list of colors in the `Colors.scss` file.
+
+# Defining Global Styles
+
+Create a `Global.scss` file to define global styles that will be used throughout the project. Import the Colors.scss file into the `Global.scss` file so that the color variables can be used in the global styles.
```
Comment (wijohnst) [inline README.md:113]: So this is one of the powers of a Sass. This pattern of being able to effectively extend a stylesheet by importing another.

File: README.md [hunk 2]
```diff
@@ -102,7 +102,24 @@ In order to import our SVGs as React components, we uuse the ReactComponent impo
...
+
+# Importing Global Styles
+
+Import the `Global.scss` file into your main SCSS file so that the global styles are applied to the entire application. You can then simply use the css function `var()` to call the color you'd like to use. For example, `var(--black)`.
+
+# Using Storybook
+
+Global styles will not be applied to the Storybook stories by default.
+To make the global styles available in Storybook, add the import statement for the `Global.scss` file to the `preview.js` file in the Storybook configuration. This ensures that the stories are consistent with the rest of the application.
```
Comment (wijohnst) [inline README.md:122]: Another great example of Storybook configuration. We can identify the following pattern when working with `Storybook`:

1. Adding a new global dependency (in this case `Global.scss`) 
2. Updating `Storybook` to handle that global dependency

I'm not 100% sure why I'm calling this out, other than I didn't really think about this kind of stuff until later in my growth as a dev. And it feels valuable for some reason.

At any rate, great explanation. And great job figuring all this out. Lots of good stuff here.

File: apps/web/.storybook/GlobalDecorator.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { addDecorator } from '@storybook/react';
+import '../src/Styles/Global.scss';
+
+const GlobalDecorator = (storyFn) => (
+  <div className="global-wrapper">{storyFn()}</div>
+);
+
+addDecorator(GlobalDecorator);
```
Comment (wijohnst) [inline apps/web/.storybook/GlobalDecorator.tsx:9]: Be careful! 

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md?ref=storybookblog.ghost.io#deprecated-addparameters-and-adddecorator

If you hover over `addDecorator` in VSCode does it say that the method is deprecated after v6.0?

File: apps/web/.storybook/preview.js [hunk 4]
```diff
@@ -1,3 +1,8 @@
+import { configure } from '@storybook/react';
+import './GlobalDecorator';
+
+configure(require.context('../src', true, /\.stories\.ts$/), module);
```
Comment (wijohnst) [inline apps/web/.storybook/preview.js:4]: What does this do?

File: apps/web/src/Styles/Colors.scss [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  /* Light Mode */
+  --black: #1c1c1c;
+  --blue: #3478cc;
+  --whitesmoke: #dbe0e5;
+  --gray: #888a8c;
+  --orange: #eb5428;
+  --white: #ffffff;
+
+  /* Dark Mode */
+  --black-dark: var(--white);
```
Comment (wijohnst) [inline apps/web/src/Styles/Colors.scss:11]: Are these just placeholder values for now?

**Review by wijohnst — COMMENTED** [review]
Really really great work on this.

**Review by wijohnst — APPROVED** [review]
Looks good.
